### ISSUE TITLE
Naming consistency of YAML keys

### ISSUE THREAD

**Issue by yoichitgy** [Swinject/Swinject-CodeGeneration#29]
Swinject defines some terms as below.
- **Service:** A protocol defining an interface for a dependent type.
- **Component:** An actual type implementing a service.
- **Factory:** A function or closure instantiating a component.
- **Container:** A collection of component instances.

https://github.com/Swinject/Swinject/blob/master/Documentation/DIContainer.md

It's good to follow the same naming convention instead of `baseClass` or `targetClass`.

**Comment by yoichitgy** [comment]
Also, in YAML files, shorter keys might be preferable because users have to repeat typing the keys again and again. String keys might be better than symbol keys to reduce typing a `:`.

For example,

```
  - args:
    - arg:
      - name: argumentName
      - type: ArgumentType
    - arg:
      - name: title
      - type: String
```

or

```
  args:
  - argName: argumenttypewithoutspecificname
    argType: ArgumentTypeWithoutSpecificName
  - argName: title
    argType: String
```

@Lutzifer what do you think?

**Comment by Lutzifer** [comment]
I agree on using shorter names/strings instead of symbols, feel free to adapt it.
We only have csv in use in our projects atm.
I would like to keep the overall structure, as it is the same that is used inside the script, so this makes yml support a breeze.

**Comment by yoichitgy** [comment]
I got it about YAML structure. It's good to keep the structure as you told for efficiency😃

**Comment by yoichitgy** [comment]
Closed by #39.
